src/index.ts

```diff
@@ -1,28 +1,33 @@
-import '/build/css/style.css'
-import { Movie_Search, Result } from './Movie'
-import { api_key } from './key.js'
+import { MovieSearch, Result } from './Movie'
+import { apiKey } from './key'
 
-const movieDisplay = document.getElementById('movie-display-container')!
+const movieDisplay = document.getElementById(
+  'movie-display-container'
+) as HTMLElement
 const movieSearchBox = document.getElementById(
   'movie-search-box'
 ) as HTMLInputElement
 
 // Get Search Term to call API
 function searchMovies(searchTerm: string): void {
   if (searchTerm.length > 0) {
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises, @typescript-eslint/no-use-before-define
```

Comment:
I see you made https://github.com/tyseawood/Flick-fil-a/issues/18 to track fixing this lint issue and the one on line 22. That's totally fine since there are only two, but I'll just highlight what I might do in a more complex project:

I would create a new issue (number 19 in our case) to "Fix floating promise" and then above the lint comment I would add a TODO like:
```javascript
if (searchTerm.length > 0) {
      // TODO(https://github.com/tyseawood/Flick-fil-a/issues/19): Fix floating promise.
      // eslint-disable-next-line @typescript-eslint/no-floating-promises, @typescript-eslint/no-use-before-define
      loadMovies(searchTerm)
}
```

And then in issue 19 I would have notes for my future self on how to fix it (like maybe if you were reading any articles when you were trygin to debug you could link them or something). 

Anyways not important here per se but just giving you an example for the future. 

I also see your PR title is "XX: fix thing". You don't have to number the PR titles. You should be able to tag the corresponding issues in the description with like "fixes #18" or just via the UI (I haven't tested it but I remember it working like that).

Reply:
I've been creating the pr's from the branches generated directly from the issues and that's what they're named by default, but I can start renaming them to make more sense.

Reply:
oh gotcha that makes sense. it's not critical